support-workers/build.sbt

```diff
@@ -49,14 +49,34 @@ IntegrationTest / assembly / assemblyMergeStrategy := {
 IntegrationTest / assembly / test := {}
 assembly / aggregate := false
 
-lazy val deployToCode =
-  inputKey[Unit]("Directly update AWS lambda code from CODE instead of via RiffRaff for faster feedback loop")
+// We also have to put the build number in the .jar, because AWS refuses to create a new lambda version if the jar is the same!
+Compile / resourceGenerators += Def.task {
+  val buildNumber = lambdaVersion.value
+  val file = resourceManaged.value / "build.number"
+  IO.write(file, buildNumber)
+  Seq(file)
+}.taskValue
+
+lazy val lambdaVersion = taskKey[String]("unique value to allow us to create a new AWS lambda version every time")
```

Comment:
sbt lets us create a "task" which is executed for each command, but it's shared between all the steps, this means every time we run deployToCodeWithCfn, it uses a new value across all stages of the build